sql/parse/parse.go

```diff
@@ -558,6 +593,10 @@ func convertAlterTable(ctx *sql.Context, ddl *sqlparser.DDL) (sql.Node, error) {
 	}
 }
 
+func tableNameToUnresolvedTable(tableName sqlparser.TableName) *plan.UnresolvedTable {
```

Comment:
If we're moving this to a function, I feel that all of the `plan.NewUnresolvedTable` calls should move to this function. Otherwise this one place uses it, and every other location in this same file uses the other.

Reply:
Yeah, good point. I was just being lazy I think. Done.